server/modules/allocations.js

```diff
@@ -65,10 +65,10 @@ export async function findPartnerById(partnerId) {
  * @returns {Promise} - Promise which resolves to a list of placements
  * from the past numberOfDays provided, or throws an error if unsuccessful
  */
-export const fetchNewPlacements = async (status, numberOfDays = 1) => {
-  const response = await axios.get(`api/v1/placements?status=${status}`);
+export const fetchNewPlacements = async (status) => {
+  const response = await axios.get(`http://localhost:8000/api/v1/auto/placements?status=${status}`);
   const placements = response.data.values;
-  const currentDate = new Date(Date.now());
-  const fromDate = currentDate.setDate(currentDate.getDate() - numberOfDays);
-  return placements.filter(data => Date.parse(data.created_at) >= fromDate);
+  const fromDate = new Date(new Date() - 20000); // Current date minus time interval.
```

Comment:
Can you replace this `2000` seconds with `TIME_INTERVAL` set in the environment variable?